### DISCUSSION TITLE
ruckus/R730Unleashed

### DISCUSSION THREAD

**Discussion by giscus** [ms264556/Hackery#13] [Comments]
# ruckus/R730Unleashed

Hackery

https://ms264556.net/ruckus/R730Unleashed

**Comment by EkimNosredna** [comment]
Anybody asked for a way to undo this if they want to reconnect it to a ZD or vSZ?

**Comment by ms264556** [comment]
You need to install an older release of Unleashed, [use this to get a root shell](https://ms264556.net/ruckus/UnleashedAddRootShell), then in the root shell set the model and name back to R730, then upgrade to an R730 solo firmware. I can add a guide for this over the Easter break.

**Reply by EkimNosredna** [reply]
Thank you much, I'll give it a shot for now though. wish they would just allow it to be run I know the AP was kinda an oops but still... we have a hundred+ we are about to replace... depending on certain other factors...

**Reply by EkimNosredna** [reply]
Side note I'm really bad at writing, and I tried one AP while I was waiting to see if you would respond, I ended up getting the AP to load the firmware that this article uses at the top, tried to set the BSP back didn't think it worked... tried something else, and then tried to start with what you were saying on the other AP I experimented on, and logged into my ZD and noticed that the first one was sitting waiting to be approved...
It seems to be pushing the ZD firmware now...

**Comment by ms264556** [comment]
So long as you have a ZoneDirector running a release which supports the R850 then I think there should be no ill effects.

It's just nice to set it back to R730 in case someone tries to run SmartZone 6.1.1+ or Unleashed 200.15+ or an older ZD release, since these are incompatible.

**Reply by EkimNosredna** [reply]
Yeah, in my case, I still have a couple of R700's on this ZD so I'm as far forward as I can go until I replace those...

**Comment by sayantangr** [comment]
Hello 

160mhz working or unstable?

**Reply by ms264556** [reply]
As I noted in the webpage, 160MHz doesn't seem to play nice with other nearby APs/routers. This is probably why it's disabled on all devices with the same chipset.

Incidentally, many (?most?) 802.11ax client devices which support 80MHz 2x2 only support 160MHz 1x1, so that they will not see any extra throughput from enabling 160MHz.

**Comment by codykl** [comment]
I can't seem to update much past version 200.15.6.12.304, I've tried 200.15.6.112.54, 200.15.6.212.20. gets to upgrading 99% and then has a "deploy timeout", I've tried after a factor reset and after a setup as a dedicated master, but it does not work. Maybe there was an additional check added in the newer firmware?

**Reply by ms264556** [reply]
Ruckus browser support is a dumpster-fire. If you've been using Firefox (since this is the only browser which works on older Ruckus firmware) then you may need to switch to Chrome/Edge to upgrade on newer Ruckus firmware releases.

I do a quick R730 upgrade check after every new release, and e.g. 200.17 upgraded OK. I think 200.18 is imminent, so I'll re-do my checks *including the latest 200.15* once 200.18 is released.

If all else fails then [apfw_upgrade](https://docs.commscope.com/bundle/unleashed-200.15-commandref/page/GUID-5E429799-F6D6-4314-840C-A4469916FCB6.html) from the CLI will give you much better error information. If this reports a tail signature failure then you can usually get the upgrade done by switching into [ap-mode](https://docs.commscope.com/bundle/unleashed-200.17-commandref/page/GUID-98BB49DE-3E1E-48F9-BDAF-764A42BEE1F1.html) and using the [fw](https://support.ruckuswireless.com/articles/000001551) command to upgrade.

**Reply by codykl** [reply]
I was able to do the update to 200.15.6.212.20 in ap-mode with fw command. Thanks. fyi 200.18 is out as of today

**Reply by wisecs** [reply]
Are y'all saying that you were able to upgrade the R730 to 200.15?  That seems worth it for the security updates

**Comment by weijersnetwerken** [comment]
If someone attempts to revert the units, here’s my experience in case it might help someone: 

I performed this process on four R730s, reverting all of them after acquiring R850s.

Initially, I factory-reset the units using the Unleashed GUI. After the reset, I individually configured each device in Unleashed, which then allowed me to start the reversion process.

Following the reversion, I had to log into the solo firmware using the credentials from Unleashed, as the default credentials (super/sp-admin) didn’t work. However, the units still displayed as R850 instead of R730, and I was unable to upgrade or access any configuration options—every relevant feature was missing from the web GUI, despite trying multiple browsers.

To resolve this, I performed another factory reset on each unit. Only after this final reset were the devices fully reverted. This occurred across all four units.

**Comment by EkimNosredna** [comment]
You can also follow the steps to make it an R850 and just set it back to an
R730... but thanks for this it might let me get back into one I screwed
up... otherwise I have a usb serial to pin headers that I'm going to drop
on the jtag port and try to manually do it that way.

**Comment by RikiTheHeropon** [comment]
Great guide, thank you! My R730 is performing great after flashing it with Unleashed.

There's just one quirk I've noticed where max clients under AP info is stuck at 100; this is despite setting it higher under model specific R850 settings, as well as increasing max clients per radio under WLAN settings. Is this just a visual bug?

**Reply by ms264556** [reply]
An Unleashed Master AP only supports 100 clients.

You can see this mentioned e.g on page 6 of the [Unleashed Datasheet](https://www.ruckusnetworks.com/globalassets/digizuite/960644-ruckus-unleashed-portfolio-datasheet.pdf) (in a footnote).

**Reply by ms264556** [reply]
(and glad the guide helped!)

**Reply by RikiTheHeropon** [reply]
Whoops, I'm still learning about Ruckus and Unleashed so that's good to know. Thanks for the quick response!

**Comment by EkimNosredna** [comment]
Trying to recover my dead AP and the second factory reset didn't work as far as I can tell.
Anyone have access to the https://florisbrunet.com/blog/recovering-a-ruckus-access-point-using-the-serial-port/ and the pin out and settings required to access the CLI? I'm trying different baud's and only getting gibberish...

disregard... internet archive for the win: https://web.archive.org/web/20250130144919/https://florisbrunet.com/blog/recovering-a-ruckus-access-point-using-the-serial-port/

**Comment by midnextlevel** [comment]
Hello i have a T710 as unleased master, will this work?

**Reply by ms264556** [reply]
If you're asking "can I run R730 and T710 APs on the same Unleashed network?" then yes, you can. But they both need to run the same version of Unleashed, so [you'll need to downgrade your T710 to Unleashed 200.14 ](https://ms264556.net/ruckus/UnleashedFirmwareGuide?q=R730+T710)if it's running a 200.15 release.

**Reply by midnextlevel** [reply]
Thanks thanks and thank you

**Comment by davecsmith** [comment]
I got my 730 to the unleashed firmware following this, but now when I try to configure it it just seems to lock up. Any tips? If I reboot I can redo the wizard, and it gets to the final step and stops responding...I can still ping, and refresh the page, but nothing else.

**Reply by ms264556** [reply]
1st, [make sure you're using a supported release](https://ms264556.net/ruckus/UnleashedFirmwareGuide?q=R730). In theory you can install versions 200.9 - 200.14, but I only tested 200.13 & **200.14**.

2nd, assuming you're using a supported release, try a different browser.
* Older Ruckus firmware requires Firefox, otherwise various wizards will not complete correctly.
* Very recent Ruckus firmware requires Chrome/Edge otherwise upgrade wizards won't complete correctly.

So it's always a good idea to switch between Edge/Chrome <=> Firefox if some wizard is hanging, or some process isn't taking effect.

**Comment by davecsmith** [comment]
Thanks, Tony

So I ended up doing it from the cli, and then once i got past the wizard
everything appears to work fine.

**Comment by answero453** [comment]
Quick question. In the AP group settings, you can select Model Specific Control in the Other tab. By default, it shows the H350. I tried to choose the R850, but those settings are not saved.

**Comment by alclary** [comment]
Thanks for the awesome guide!

I'm facing an issue with supplying power via PoE to the R730 while also getting it to negotiate higher than 100Mbps when running data and power on the PoE port. Running unleashed per the guide above. 

Following your notes above, I have the R730's 'PoE Operating Mode' set to 802.3bt/class5. The R730 can certainly get enough power over the PoE port with this mode, as when I run my data cable to the non-PoE eth port I am getting full speeds and full power on the device. The issue is that when PoE is being supplied via the PoE eth port, the R730 refuses to negotiate higher than 100Mbps. In the cli, the ap-mode setting won't even offer an option beyond 10Mb, 100Mb, or auto.

My PoE injector is allegedly rated for 802.3bt, so in theory I would expect all 4 pairs to be in use for both power and data, and for it to be possible for 1000Mbps to be negotiated. I am also running just a few feet of Cat6 cable for testing and the cable transmits at 1000Mbps just fine when connected as a data line alone. Do you have any advice or experience regarding the R730 that might help illuminate the issue here?

**Reply by ms264556** [reply]
Damaged or poorly terminated cabling will cause a Ruckus AP to quickly train down to 100Mbps.

But assuming you've swapped out *all* the cables between the switch and your AP then my guess would be your R730 is a little damaged.

I believe a not-insignificant proportion of Ruckus APs sold on eBay are RMA units (since Ruckus often asks you to dispose of the defective unit). I have a donated R730 with a damaged PoE chip & it exhibits similar issues.

Either return it for a replacement, or use the PoE port for power use the 1Gb port for data.

**Comment by yamvmax** [comment]
Currently have 2 r730's I acquired. Can i flash both with R850 FW, and use both of them with wireless enabled? Or is this hack just to turn one of the r730's into a controller ONLY? Thanks

**Reply by ms264556** [reply]
Yes, sure you can run with wireless enabled.

**Comment by yonghu996** [comment]
Hello, thank you very much for sharing the tutorial. I’ve run into a problem and would appreciate your advice.

After converting my R730 into an R850, I encountered some issues. I followed the tutorial at https://ms264556.net/ruckus/R730Unleashed#reverting-to-r730-solo-firmware to revert it to the R730 Solo firmware.

However, after restoring the R730 information with the following commands:

```text
ruckus$ bsp set name R730
ruckus$ bsp set model R730
ruckus$ bsp commit
Saving flash .....
bdSave: sizeof(bd)=0x7c, sizeof(rbd)=0xd0
  caching flash data from /dev/mtd16 [ 0x00000000 - 0x00010000 ]
  updating flash data [0x00000000 - 0x0000007c] from [0x7e9cdaec - 0x7e9cdb68]
  updating flash data [0x00008000 - 0x000080d0] from [0x7e9cdb68 - 0x7e9cdc38]
_erase_flash: offset=0x0 count=1
Erasing 64 Kibyte @ 0 -- 100 % complete
  caching flash data from /dev/mtd16 [ 0x00000000 - 0x00010000 ]
  verifying flash data [0x00000000 - 0x0000007c] from [0x7e9cdaec - 0x7e9cdb68]
  verifying flash data [0x00008000 - 0x000080d0] from [0x7e9cdb68 - 0x7e9cdc38]
... Changes saved to flash
ruckus$ client_loop: send disconnect: Connection reset
```

the connection was unexpectedly interrupted. I can no longer connect to the R730. Is there any way to recover it?

**Reply by gardar** [reply]
Have you tried resetting the AP with the reset button? In my experience you should be able to boot the 200.14 R850 firmware even if you have restored the model and name to R730.

If the R730 does not come online at all then you can connect to the serial console of it - it's pretty easy.
There's a serial header on the R730 that you can use with a TTL UART adapter at 115200 baud. Just connect to the TX, RX and GND and leave the 3.3 disconnected..
```
   1    2    3    4    5
  TX   N/C  3.3V GND   RX
```
It would be interesting to know where in the bootup it gets stuck.
The R730 has two firmware images/partitions and if you upgraded from R730_10.1.2.0.120.bl7 to R850_200.14.6.1.203.bl7 and did not do any further upgrades you should have the R730 firmware as one image and the R850 firmware on the other. In that case the simplest fix would be to enter uboot (`ctrl+c` during bootup) and then switch over to the other image with `bootid 1` or `bootid 2` followed by `reset`

If neither image slots work then there are other ways to recover it, such as live booting a firmware image, etc.
